render-test/render_test.cpp

```diff
@@ -101,9 +101,6 @@ ArgumentsTuple parseArguments(int argc, char** argv) {
         mbgl::Log::Info(mbgl::Event::General, stream.str());
         mbgl::Log::Error(mbgl::Event::General, e.what());
         exit(2);
-    } catch (const std::regex_error& e) {
```

Comment:
why is this change?

Reply:
Initially, args::ValueFlag<> filter flag was using regexp as it's type, now, conversion is done in a different place, and `} catch (const std::regex_error& e) {` is never executed.